TouchSky/TouchSky/TouchSky/Views/ListTab.swift

```diff
@@ -11,23 +11,26 @@ import SlidingTabView
 struct ListTab: View {
     @State private var tabIndex = 0
     var body: some View {
-        VStack{
-            SlidingTabView(selection: $tabIndex, tabs: ["All", "Completed", "To Do"], animation: .easeInOut)
-            Spacer()
-            if tabIndex == 0 {
-                    PeakList()
-            } else if tabIndex == 1 {
-                Text("Completed")
-            } else if tabIndex == 2 {
-                Text("To Do")
+        NavigationStack {
+            VStack{
+                SlidingTabView(selection: $tabIndex, tabs: ["All", "Completed", "To Do"], animation: .easeInOut)
+                Spacer()
+                if tabIndex == 0 {
+                    AllList()
```

Comment:
So, if you did want to pass in filtered peaks into a shared List view, you could do that here by doing something like:

`PeakList(peaks: ModelData.allPeaks)`
`PeakList(peaks: ModelData.completedPeaks)`
`PeakList(peaks: ModelData.toDoPeaks)`